source/_integrations/leviosa_shades.markdown

```diff
@@ -0,0 +1,49 @@
+---
+title: Leviosa Motor Blinds Zone hub
+description: Instructions on how to integrate the Leviosa Zone hub into Home Assistant.
+ha_category:
+  - Cover
+ha_iot_class: Local Push
+ha_release: 2021.4
```

Comment:
Release should be 2023.11 if everything goes well

Reply:
I have submitted an update